### DISCUSSION TITLE
in app chat

### DISCUSSION THREAD

**Discussion by yim0000** [PlayCover/PlayCover#1140] [Q&A] [answered]
I was playing genshin impact and i recently updated to sonoma. since 2.05 doesn't work anymore due to controls issues, i had to use 3.00 and i was unable to type in the in game chat like before, is there a way to get rid of the controls? isn't the alt button supposed to snap the controls off so I can type and click stuff?

**Comment by TheMoonThatRises** [comment, marked as answer]
Go into PlayCover and turn off Smart Keymap for the app.
